src/main/resources/default/templates/biz/cluster/cluster.html.pasta

```diff
@@ -1,137 +1,264 @@
+                            <i:local name="jobInfo"
+                                     value="@node.getJobs().get(job)"/>
+                            <i:if test="jobInfo != null">
+                                <div class="mb-4 d-flex flex-row">
+                                    <div class="mr-auto">
+                                        <div class="d-flex flex-row">
+                                            <t:dot color="@jobInfo.isLocalOverwrite() ? 'red' : 'green'"><b>@node.getNodeName()</b>
+                                            </t:dot>
+                                            <i:switch test="@jobInfo.getSynchronizeType().toString()">
+                                                <i:block name="LOCAL">
+                                                    <t:tag color="blue">@jobInfo.getSynchronizeType()</t:tag>
+                                                </i:block>
+                                                <i:block name="CLUSTER">
+                                                    <t:tag color="violet">@jobInfo.getSynchronizeType()</t:tag>
+                                                </i:block>
+                                                <i:block name="DISABLED">
+                                                    <t:tag color="yellow">@jobInfo.getSynchronizeType()</t:tag>
+                                                </i:block>
+                                            </i:switch>
+                                        </div>
+                                        <div class="muted small">@jobInfo.getExecutionInfo()</div>
+                                    </div>
+                                    <div>
+                                        <i:if test="jobInfo.isLocalOverwrite()">
+                                            <a href="/system/cluster/local/enable/@node.getNodeName()/@job"
+                                               class="btn btn-outline-primary btn-sm">
+                                                <i class="fa fa-check"></i><span
+                                                    class="d-none d-xl-inline"> Enable</span>
+                                            </a>
+                                            <i:else>
+                                                <a href="/system/cluster/local/disable/@node.getNodeName()/@job"
+                                                   class="btn btn-outline-secondary btn-sm">
+                                                    <i class="fa fa-power-off"></i><span
+                                                        class="d-none d-xl-inline"> Disable</span>
+                                                </a>
+                                            </i:else>
+                                        </i:if>
+                                    </div>
+                                </div>
+                            </i:if>
+                        </i:for>
```

Comment:
Is it possible to extract some of this into an external template? It seems to be duplicated in both modes